andela/AndelaEatsUI
# #162008105 Implement isActive status

#### What does this PR do?
Implement isActive status on a request when adding or updating a vendor

#### Description of Task to be completed?
Fix endpoints for adding and updating vendors

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `fix-endpoints-add-update-vendors-162008105`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendors`, and click on `Add Vendor` 
- Fill the form and submit a request
- run `npm test` to ensure all tests are passing


#### What are the relevant pivotal tracker stories?
- [#162008105](https://www.pivotaltracker.com/n/projects/2178749)

#### Background Context
Nil

#### Screenshots (If Applicable)
Nil
